strings-csharp/HexToByteArray/HexToByteArray/Program.cs

```diff
@@ -0,0 +1,22 @@
+﻿// See https://aka.ms/new-console-template for more information
+
+using BenchmarkDotNet.Running;
+using HexToByteArray;
+
+string hexString = "FF008000"; // Hex representaion of color System.Drawing.Color.Green
+
+var convertHex = new ConvertHex();
+
+// Run the built-in first and display the result
+var bytes = convertHex.FromHexString(hexString);
+foreach (var b in bytes)
```

Comment:
This could be extracted to a local helper method, something like "PrintArray". It might also be nice to add some simple formatting. Something like "{1,2,3,4}" so that the array output is more readable. You could use the string.Join method to comma separate the array.

Reply:
Done. Thanks!